src/Tribe/Service_Providers/Context.php

```diff
@@ -44,32 +45,34 @@ public function register() {
 	 * @return array The modified context locations.
 	 */
 	public function filter_context_locations( array $locations = [] ) {
-		$locations = array_merge(
+		$view_query_var = View::query_var();
+
+		$locations      = array_merge(
 			$locations,
 			[
-				'event_display'        => [
+				'event_display'      => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 8 space(s) between "'event_display'" and double arrow, but found 6.

<sub>hash: d8eb274b47dd55bac9f590b2168885d4</sub>